tests/integration_tests/test_cli.py

```diff
@@ -106,6 +127,22 @@ def test_check_configs(self):
             cli.check_config(config)
 
     def test_overview( self ):
+        wallet: 'bittensor.Wallet' = self.init_wallet()
+        bittensor.subtensor.register = MagicMock(return_value = True)  
+        
+        config = self.config
+        config.wallet.path = wallet.path
+        config.command = "overview"
+        config.subtensor._mock = True
+        config.subtensor.network = "mock"
+        config.no_prompt = True
+
+        cli = bittensor.cli(config)
+        cli.run()
+
+    def test_overview_no_wallet( self ):
+        # Delete wallet if exists
+        self.rm_wallet()
```

Comment:
Rather than calling `rm_wallet`, why not modify the config such that `config.wallet.path` points to an empty directory? When we run tests in parallel there's no guarantee that this will run at the same time as another test that relies on an extant wallet and mess with the results in a sporadic way.

Reply:
this is from some other code that does wallet stuff in the wallet tests. Instead I can just mock the IO, if you think that makes more sense.

Reply:
Mocked the IO https://github.com/opentensor/bittensor/pull/757/commits/adb54cf96387f546143966f658a3efee9d13f74b